src/internal/analytics-metadata/dom-utils.ts

```diff
@@ -35,3 +35,14 @@ export const isNodeComponent = (node: HTMLElement): boolean => {
     return false;
   }
 };
+
+export function findByClassNameUp(node: HTMLElement | null, className: string): HTMLElement | null {
+  if (!node || !className) {
```

Comment:
it shouldn't, but I thought it was better to still check

Reply:
I tried to remove the check, and `element.matches` throws a syntax error when the selector is `''`.

Reply:
But the selector is not supposed to be empty - it is only empty in the tests which means the tests are created for an non-existing use case. I suggest removing the check and update the tests to use some `.dummy` selector instead of an empty string.

Reply:
Done